en/drafts/originals/clustering-visualizing-word-embeddings.md

```diff
@@ -90,32 +85,32 @@ Critically, whereas we used to need as many columns as there were words to repre
+However, a more sophisticated method is to adapt word2vec to recognise that the distribution of terms in a corpus is meaningful in ways *beyond* their immediate context. Doc2vec is this extension: it adds a 'paragraph vector' that represents the source text as a whole and which is trained in parallel with the word embeddings. In a sense, this implements the intuition that we *also* see in Latent Dirichlet Allocation (LDA) that a paragraph is *about* something — a topic — that subtly influences word choice and usage. In this tutorial we found that doc2vec delivered more readily interpretable results.
 
 ## Case Study: E-Theses Online
 
-This tutorial uses a bibliographic corpus managed by the [British Library](https://www.bl.uk/) (BL) to enhance public access to doctoral research. [*E-Theses Online*](https://ethos.bl.uk/) (EThOS, hereafter) provides metadata—which is to say, data *about* a PhD, *not* the data *of* the PhD—such as Author, Title, Abstract, Keywords, [Dewey Decimal Classification](https://en.wikipedia.org/wiki/Dewey_Decimal_Classification) (DDC), etc. Most users of EThOS will search this metadata using the BL's web interace to find and retrieve individual documents (see Figure 1); however, in aggregate, [the data](https://bl.iro.bl.uk/catalog?locale=en&q=%22UK+Doctoral+Thesis+Metadata+from+EThOS%22&search_field=all_fields&sort=year_published_isi+desc&_ga=2.85833567.757719569.1646905901-495121888.1646905901) also provide a unique perspective on U.K. Higher Education.
+This tutorial uses a bibliographic corpus managed by the [British Library](https://www.bl.uk/) (BL) to enhance public access to doctoral research. [*E-Theses Online*](https://ethos.bl.uk/) (EThOS, hereafter) provides metadata — which is to say, data *about* a PhD, *not* the data *of* the PhD—such as Author, Title, Abstract, Keywords, [Dewey Decimal Classification](https://en.wikipedia.org/wiki/Dewey_Decimal_Classification) (DDC), etc. Most users of EThOS will search this metadata using the BL's web interface to find and retrieve individual documents (see Figure 1); however, in aggregate, [the data](https://bl.iro.bl.uk/catalog?locale=en&q=%22UK+Doctoral+Thesis+Metadata+from+EThOS%22&search_field=all_fields&sort=year_published_isi+desc&_ga=2.85833567.757719569.1646905901-495121888.1646905901) also provide a unique perspective on U.K. Higher Education.
 
-{% include figure.html filename="EThOS.png" caption="EThOS Web Interface for an Individual Metadata Record" %}
+{% include figure.html filename="EThOS.png" alt="Visual description of figure image" caption="Figure 1. EThOS Web Interface for an Individual Metadata Record" %}
```

Comment:
As part of our commitment to accessibility, we are now asking authors to provide descriptive alt-text to accompany figure images.

[This resource](https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81) may be useful. It advises that the alt-text for graphs and data visualisations should consist of the following:

`alt="Chart type of data type where reason for including chart”`

You can either add the alt-text into the template I've created here, or leave a comment and I’ll add it for you.

Reply:
Incoming commit should address this.